src/3D/Camera.cpp

```diff
@@ -9,28 +9,97 @@
 
 #include "Camera.h"
 #include "3D/LandIsland.h"
+#include "ECS/Components/Transform.h"
 #include "ECS/Registry.h"
 #include "Game.h"
 
 #include <glm/gtx/euler_angles.hpp>
+#include <glm/gtx/intersect.hpp>
+#include <glm/gtx/norm.hpp>
+
+#define OUT
```

Comment:
?

Reply:
stylistic choice to make out params very visible in the function declaration, can be removed if you really dislike